### ISSUE TITLE
It seems that x_gcn and new_x are not combined properly in UnionLayer

### ISSUE THREAD

**Issue by bravelywangexocr** [wenwenyu/PICK-pytorch#27]
Hi wenwenyu,

It appears that the padding in the T dimension is not considered for x_gcn before adding with new_x. 

https://github.com/wenwenyu/PICK-pytorch/blob/593c2644040da71bd04e6a9029c01464a5b83a8c/model/decoder.py#L171

Can you help confirm this? 

Thanks

**Comment by wenwenyu** [comment]
Hi,

Thanks for your mention.  I have checked it again and the way of combining x_gcn and new_x is not error or typos. We want to use x_gcn to aid new_x in the BiLSTM layer at every time step (T dimension). In the beginning, we tried to use x_gcn as the initial hidden state of BiLSTM instead of combining x_gcn with new_x at every time step. but the performance hasn't improved. This way of the combine can improve performance.
